x/stakeibc/keeper/deposit_records.go

```diff
@@ -39,7 +41,7 @@ func (k Keeper) TransferExistingDepositsToHostZones(ctx sdk.Context, epochNumber
 		return isTransferRecord && isBeforeCurrentEpoch
 	})
 
-	ibcTimeoutBlocks := k.GetParam(ctx, types.KeyIbcTimeoutBlocks)
+	feeTransferTimeoutNanos := k.GetParam(ctx, types.KeyFeeTransferTimeoutNanos)
```

Comment:
do you think we should raise the timeout here from 10 minutes? Given epochs are 6 hours, it might okay to make the timeout 30-60m?

Reply:
definitely! no reason to make this timeout so low. done.